lib/cpp/test/TSSLSocketInterruptTest.cpp

```diff
@@ -91,8 +90,6 @@ BOOST_GLOBAL_FIXTURE(GlobalFixtureSSL);
 BOOST_GLOBAL_FIXTURE(GlobalFixtureSSL)
 #endif
 
-BOOST_FIXTURE_TEST_SUITE(TSSLSocketInterruptTest, TestPortFixture)
```

Comment:
Can you keep 3 of these as BOOST_**AUTO**_TEST_SUITE ?

This would remove a namespace and put everything into global namespace.
In this regard, it seems that this one should have been placed before the `keyDir` symbol already...